en/db-models-relationships.md

```diff
@@ -334,6 +334,24 @@ class Invoices extends Model
                 ]
             ]
         );
+
+        $this->hasMany(
+            'inv_cst_id',
+            Customers::class,
+            'cst_id',
+            [
+                'reusable' => true,
+                'alias'    => 'customersInactive',
+                'params'   => function() {
```

Comment:
I would add here some comment, explaining for what purposes.

Reply:
I modified the example and added an extra explanation.